Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.11
+
+ADD requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+RUN echo > /root/.bashrc
+RUN echo "PS1='🐳\[\033[01;32m\]\u@:\[\033[01;34m\]\W\[\033[00m\] \$ '" >> /root/.bashrc
+RUN echo "alias ls='ls --color=auto'" >> /root/.bashrc
```

Comment:
What's this doing? Can we add a comment to explain these lines?

Reply:
The first line modifies your prompt, the second makes `ls` by default have colors for folders and files. I will add a comment.

<img width="1182" alt="Screenshot 2023-08-01 at 09 10 23" src="https://github.com/skiller-whale/core-python/assets/3115248/dc027a4c-1a10-4faf-8abc-2c196174f639">

Reply:
Is this standardised across our images? Or just for this Python one?

Reply:
That's a good point. There's actually nothing in the other images.

I suppose I can remove that from here to make it more consistent.

<img width="1182" alt="Screenshot 2023-08-01 at 09 33 06" src="https://github.com/skiller-whale/core-python/assets/3115248/58355b99-82f1-4b0b-a35d-7b1ce08f095b">